### PR TITLE
some-small-fixes

### PR DESCRIPTION
**Pull request by wissog**
<3

### CODE REVIEW

File: src/components/Sections/Library/Share.js [hunk 1]
```diff
@@ -25,10 +25,8 @@ const LibraryShare = ({ t, position }) => {
     window.addEventListener('resize', closePopup);
   }, []);
 
-  useEffect(() => {
-    return () => {
-      window.removeEventListener('resize', closePopup);
-    };
+  useEffect(() => () => {
+    window.removeEventListener('resize', closePopup);
```
Comment (gshilin) [inline src/components/Sections/Library/Share.js:29]: It **must** be returned, then it will be called on unmount of this component

Comment (gshilin) [reply src/components/Sections/Library/Share.js:29]: This and previous useEffect should be one:
```js
useEffect(() => {
    window.addEventListener('resize', closePopup);

    return () => window.removeEventListener('resize', closePopup);
  }, []);

```

File: src/components/Sections/Music/Music.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      {content}
+    </>
+  );
+}
+
+Music.propTypes = {
+  t: PropTypes.func.isRequired,
+}
+
+export default withNamespaces()(Music);
```
Comment (gshilin) [inline src/components/Sections/Music/Music.js:53]: Why you didn't apply lint on this file?

File: src/components/Sections/Programs/tabs/Clips/List.js [hunk 3]
```diff
@@ -25,13 +25,11 @@ const renderUnit = (unit, t) => {
...
-      renderUnit={renderUnit}
-      extraFetchParams={{ content_type: CT_CLIP }}
-    />
-  </div>;
+  <UnitList
+    key="programs-clips"
+    namespace="programs-clips"
+    renderUnit={renderUnit}
+    extraFetchParams={{ content_type: CT_CLIP }}
+  />;
```
Comment (gshilin) [inline src/components/Sections/Programs/tabs/Clips/List.js:33]: Are you sure the wrapping < div > was not required for CSS?

File: src/components/Sections/Programs/tabs/Programs/List.js [hunk 4]
```diff
@@ -35,13 +35,11 @@ const renderUnit = (unit, t) => {
...
-      renderUnit={renderUnit}
-      extraFetchParams={{ content_type: CT_VIDEO_PROGRAM_CHAPTER }}
-    />
-  </div>;
+  <UnitList
+    key="programs-main"
+    namespace="programs-main"
+    renderUnit={renderUnit}
+    extraFetchParams={{ content_type: CT_VIDEO_PROGRAM_CHAPTER }}
+  />;
```
Comment (gshilin) [inline src/components/Sections/Programs/tabs/Programs/List.js:43]: Are you sure the wrapping < div > was not required for CSS?

File: src/components/Sections/Publications/tabs/Articles/List.js [hunk 5]
```diff
@@ -25,14 +25,12 @@ const renderUnit = (unit, t) => {
...
-      extraFetchParams={{ content_type: CT_ARTICLE }}
-      renderUnit={renderUnit}
-    />
-  </div>
+  <UnitList
+    key="publications-articles"
+    namespace="publications-articles"
+    extraFetchParams={{ content_type: CT_ARTICLE }}
+    renderUnit={renderUnit}
+  />
```
Comment (gshilin) [inline src/components/Sections/Publications/tabs/Articles/List.js:33]: Are you sure the wrapping < div > was not required for CSS?

File: src/components/Sections/Publications/tabs/AudioBlog/Container.js [hunk 6]
```diff
@@ -19,14 +19,12 @@ const renderUnit = (unit, t) =>
...
-      extraFetchParams={{ content_type: CT_BLOG_POST }}
-      renderUnit={renderUnit}
-    />
-  </div>
+  <UnitList
+    key="publications-audio-blog"
+    namespace="publications-audio-blog"
+    extraFetchParams={{ content_type: CT_BLOG_POST }}
+    renderUnit={renderUnit}
+  />
```
Comment (gshilin) [inline src/components/Sections/Publications/tabs/AudioBlog/Container.js:27]: Are you sure the wrapping < div > was not required for CSS?

File: src/helpers/scrollToSearch/helper.js [hunk 7]
```diff
@@ -129,7 +125,7 @@ export const wrapSeekingPlace = (data, tags, fromNohtml, toNoHtml) => {
   before += data.slice(openTagP.pos, from).replace(/<p|<h/, x => `<div class="scroll-to-search" id="${SCROLL_SEARCH_ID}">${x}`);
 
   let after = data.slice(to, closeTagP.pos);
-  after += data.slice(closeTagP.pos).replace(/<\/p>|<\/h\d>/, x => x + '</div>');
+  after += data.slice(closeTagP.pos).replace(/<\/p>|<\/h\d>/, x => `${x  }</div>`);
```
Comment (gshilin) [inline src/helpers/scrollToSearch/helper.js:128]: Why ${x } and not ${x} (i.e. without space?

File: src/helpers/utils.js [hunk 8]
```diff
@@ -240,7 +240,7 @@ export const getRSSFeedByLang = language => {
   }
 };
 
-export const getRSSLinkByLang = language => 'https://feeds.feedburner.com/' + getRSSFeedByLang(language);
+export const getRSSLinkByLang = language => `https://feeds.feedburner.com/${  getRSSFeedByLang(language)}`;
```
Comment (gshilin) [inline src/helpers/utils.js:243]: you don't need space after ${

File: src/helpers/utils.js [hunk 9]
```diff
@@ -255,7 +255,7 @@ const podcastLinks = new Map([
 
 export const getPodcastLinkByLang = language => {
   const hash = podcastLinks.get(language) || 'kabbalah-media-mp3-kab-eng/id1109845884?l=iw';
-  return 'https://podcasts.apple.com/il/podcast/' + hash;
+  return `https://podcasts.apple.com/il/podcast/${  hash}`;
```
Comment (gshilin) [inline src/helpers/utils.js:258]: You don't need space after ${

File: src/redux/modules/publications.js [hunk 10]
```diff
@@ -189,8 +189,8 @@ const onSetLanguage = draft => {
   draft.publishers.byID = {};
   draft.collections     = {};
 
-  let pageNo           = draft.twitter.pageNo;
-  let total            = draft.twitter.total;
+  let { pageNo } = draft.twitter;
+  let { total } = draft.twitter;
```
Comment (gshilin) [inline src/redux/modules/publications.js:193]: combine it with previous line

File: src/components/AVPlayer/AVPlayer.js [hunk 11]
```diff
@@ -295,7 +295,7 @@ class AVPlayer extends Component {
       }
       this.sendCallbackMessage(event.data, { status: 'ok' });
     } catch (e) {
-      console.error('Error while receive external message in AVPlayer: ' + e);
+      console.error(`Error while receive external message in AVPlayer: ${  e}`);
```
Comment (gshilin) [inline src/components/AVPlayer/AVPlayer.js:298]: You don't need space after ${

File: src/components/Pages/Unit/widgets/UnitMaterials/Sources/Sources.js [hunk 12]
```diff
@@ -101,8 +101,8 @@ const Sources = ({ unit, indexMap, t, options }) => {
       return;
     }
 
-    const newFetch = selected + '#' + language;
-    if (newFetch === fetched){
+    const newFetch = `${selected  }#${  language}`;
```
Comment (gshilin) [inline src/components/Pages/Unit/widgets/UnitMaterials/Sources/Sources.js:104]: You don't need space after ${ and after ${selected

File: src/components/Pages/Unit/widgets/UnitMaterials/Transcription/Transcription.js [hunk 13]
```diff
@@ -228,12 +228,12 @@ class Transcription extends Component {
...
 
-    const ap                      = playerHelper.getActivePartFromQuery(location);
-    let { url, text: searchText } = buildSearchLinkFromSelection(language);
+    const ap                        = playerHelper.getActivePartFromQuery(location);
+    const { url, text: searchText } = buildSearchLinkFromSelection(language);
     if (!url)
       return;
     const selectedFileProps = selectedFile ? `&selectedFileId=${selectedFile.id}` : '';
-    const searchUrl         = `${url}&activeTab=${activeTab}${selectedFileProps}${!ap ? '' : '&ap=' + ap}`;
+    const searchUrl         = `${url}&activeTab=${activeTab}${selectedFileProps}${!ap ? '' : `&ap=${  ap}`}`;
```
Comment (gshilin) [inline src/components/Pages/Unit/widgets/UnitMaterials/Transcription/Transcription.js:236]: You don't need space after ${

File: src/components/Sections/Home/LatestUpdate.js [hunk 14]
```diff
@@ -25,7 +25,7 @@ import {
 const LatestUpdate = ({ unit, label, t }) => {
   const link       = canonicalLink(unit);
   const name           = unit.name ||
-    `${t('constants.content-types.' + unit.content_type)} ${t('lessons.list.number')} ${unit.name_in_collection}`;
+    `${t(`constants.content-types.${  unit.content_type}`)} ${t('lessons.list.number')} ${unit.name_in_collection}`;
```
Comment (gshilin) [inline src/components/Sections/Home/LatestUpdate.js:28]: You don't need space after ${

File: src/components/Sections/Home/SearchBar.js [hunk 15]
```diff
@@ -20,7 +20,7 @@ class MyOmniBox extends OmniBox {
   }
 
   handleFromInputChange = (value) => {
-    window.location.href = '/' + this.props.language + '/simple-mode?date=' + moment(value).format('YYYY-MM-DD');
+    window.location.href = `/${  this.props.language  }/simple-mode?date=${  moment(value).format('YYYY-MM-DD')}`;
```
Comment (gshilin) [inline src/components/Sections/Home/SearchBar.js:23]: You don't need space after both ${ and before }/

**Review by gshilin — CHANGES_REQUESTED** [review]

File: src/components/Sections/Music/Music.js (hunk 2 above)
Comment (wissog) [reply src/components/Sections/Music/Music.js:53]: i don't know, its automatic process...

File: src/components/Sections/Programs/tabs/Clips/List.js (hunk 3 above)
Comment (wissog) [reply src/components/Sections/Programs/tabs/Clips/List.js:33]: its a redundant div

File: src/components/Sections/Programs/tabs/Programs/List.js (hunk 4 above)
Comment (wissog) [reply src/components/Sections/Programs/tabs/Programs/List.js:43]: redundant div

File: src/components/Pages/Unit/widgets/Info/Info.js [hunk 16]
```diff
@@ -44,7 +44,7 @@ const filterLessons = (ct, filmDate) => {
   }
 };
 
-const makeTagLinks = (tags = [], getTagById) =>
+const makeTagLinks = (tags = [], getTagById, filteredListPath) =>
```
[the remark this answers is not in the document]
Comment (gshilin) [reply src/components/Pages/Unit/widgets/Info/Info.js:47]: Remove it please

[the remark this answers is not in the document]
Comment (wissog) [reply src/components/Pages/Unit/widgets/Info/Info.js:47]: ok...

**Pull request merged**